src/views/Home/Challange/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import { Challange } from '#types';
+
+import styles from './styles.css';
+
+interface Props {
+    className?: string;
+    challange: Challange;
+}
+
+function ChallangeView(p: Props) {
+    const {
+        challange,
+        className,
+    } = p;
+
+    return (
+        <div className={_cs(className, styles.challange)}>
+            { challange?.title }
```

Comment:
Unnecessary check

Reply:
how so? what if challange is undefined

Reply:
In the definition of props, it's required.